### PR TITLE
Todos 8

### PR DESCRIPTION
**Pull request by joeariasc**
Edit ToDo

### CODE REVIEW

File: app/actions/todos.go [hunk 1]
```diff
@@ -37,6 +37,17 @@ func SaveTodo(c buffalo.Context) error {
 	return c.Redirect(http.StatusSeeOther, "listTodoPath()")
 }
 
+func EditTodo(c buffalo.Context) error {
+	tx := c.Value("tx").(*pop.Connection)
+	todoID := c.Param("todo_id")
+	todo := models.Todo{}
+	if err := tx.Find(&todo, todoID); err != nil {
+		return c.Render(http.StatusNotFound, r.String("ToDo not Found"))
```
Comment (danoviedo91) [inline app/actions/todos.go]: c.Error(http.StatusNotFound, errors.Wrap(err, "EditTodo - todo not found"))

File: app/templates/todos/index.plush.html [hunk 2]
```diff
@@ -18,28 +18,28 @@
...
-                </div>
-            </td>
-        </tr>
+                </td>
+                <td>
+                    <%= todo.LimitDate.Format("Monday 02, January 2006") %>
+                </td>
+                <td>
+                    <div class="row mx-0 justify-content-center">
+                        <i class="text-muted fas fa-pencil-alt mx-3"></i>
```
Comment (danoviedo91) [inline app/templates/todos/index.plush.html:36]: This one should point to the edit form of the todo

**Review by danoviedo91 — CHANGES_REQUESTED** [review]

File: app/actions/todos.go (hunk 1 above)
Comment (joeariasc) [reply app/actions/todos.go]: Done!

File: app/templates/todos/index.plush.html (hunk 2 above)
Comment (joeariasc) [reply app/templates/todos/index.plush.html:36]: Fixed

**Review by danoviedo91 — APPROVED** [review]

**Pull request merged**